api/envoy/config/rbac/v2/rbac.proto

```diff
@@ -201,17 +201,21 @@ message Principal {
     // Authenticated attributes that identify the downstream.
     Authenticated authenticated = 4;
 
-    // A CIDR block that describes the downstream direct remote address.
+    // A CIDR block that describes the downstream IP.
```

Comment:
Please add docs to clarify that this address will honor proxy protocol, but will not honor XFF.